### PR TITLE
bt-627: Fix quality criteria comments

### LINKED ISSUE #627: Fix quality criteria comments
https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/src/javascript.md
ALSO: review other parts of the mobile project where we may have similar issues

Image

Image

### PR DESCRIPTION
**Pull request by likeri29**

### CODE REVIEW

File: mobile/src/bundles/common/components/slider/constants/constants.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+const sliderValues = {
```
Comment (katerynakostikova) [inline mobile/src/bundles/common/components/slider/constants/constants.ts]:
```suggestion
const SLIDER_VALUES = {
```

File: mobile/src/bundles/employer/components/candidate-card/candidate-card.tsx [hunk 2]
```diff
@@ -11,12 +11,9 @@ import { BadgeSize, TextCategory } from '~/bundles/common/enums/enums';
 import { globalStyles } from '~/bundles/common/styles/styles';
 import { type Candidate } from '~/bundles/employer/types/types';
 
+import { maxValues } from './constants/constants';
```
Comment (katerynakostikova) [inline mobile/src/bundles/employer/components/candidate-card/candidate-card.tsx]:
```suggestion
import { MAX_VALUE } from './constants/constants';
```

**Review by katerynakostikova — APPROVED** [review]

**Review by mskmee — APPROVED** [review]

File: mobile/src/bundles/common/components/slider/constants/constants.ts (hunk 1 above)
Comment (likeri29) [reply mobile/src/bundles/common/components/slider/constants/constants.ts]: What about [B29](https://github.com/BinaryStudioAcademy/quality-criteria/blob/production/src/javascript.md)?
Maybe it'd be better if I name it SliderValues and make names there in uppercase?

Comment (katerynakostikova) [reply mobile/src/bundles/common/components/slider/constants/constants.ts]: ok, then rename file as enum

**Pull request merged**